src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -150,12 +153,12 @@
       </_SupplementalTestData>
     </ItemGroup>
     <ItemGroup>
-      <SupplementalTestDataTestDir Include="@(_SupplementalTestData->'$(TestPath)/%(DestinationDir)%(Filename)%(Extension)')" />
```

Comment:
The supplemental data is copied to two locations, `$(TestPath)` and `$(OutDir)`, whereas the other items are just copied to `$(TestPath)`.